carma_wm_ros2/include/carma_wm_ros2/SignalizedIntersectionManager.hpp

```diff
@@ -0,0 +1,160 @@
+#pragma once
```

Comment:
Take a look at Misheel's PR here
https://github.com/usdot-fhwa-stol/carma-platform/pull/1638/files

Apply any changes to the carma_wm package to this PR as well. The exception being for any logs/prints which are clearly for debugging. 

Perform this change in a single isolated commit so that I can easily double check the changes exist.

Reply:
Just to confirm, these would be the changes corresponding to #1660 ?